asyncsleepiq/fuzion/foundation.py

```diff
@@ -17,25 +17,25 @@
     Speed,
 )
 from ..foundation import SleepIQFoundation
-from ..preset import SleepIQPreset
+from .preset import SleepIQFuzionPreset
 from .actuator import SleepIQFuzionActuator
 from .light import SleepIQFuzionLight
 
 FEATURE_NAMES = [
-    "pressureControlEnabledFlag"
-    "articulationEnableFlag"
-    "underbedLightEnableFlag"
-    "rapidSleepSettingEnableFlag"
-    "thermalControlEnabledFlag"
-    "rightHeadActuator"
-    "rightFootActuator"
-    "leftHeadActuator"
-    "leftFootActuator"
-    "flatPreset"
-    "favoritePreset"
-    "snorePreset"
-    "zeroGravityPreset"
-    "watchTvPreset"
+    "pressureControlEnabledFlag",
+    "articulationEnableFlag",
+    "underbedLightEnableFlag",
+    "rapidSleepSettingEnableFlag",
+    "thermalControlEnabledFlag",
+    "rightHeadActuator",
+    "rightFootActuator",
+    "leftHeadActuator",
+    "leftFootActuator",
+    "flatPreset",
+    "favoritePreset",
+    "snorePreset",
+    "zeroGravityPreset",
+    "watchTvPreset",
     "readPreset",
 ]

```

Comment:
This array was being created with not the right number of elements due to the missing commas